stored_requests/events/http/http.go

```diff
@@ -94,10 +95,33 @@ func (e *HTTPEvents) refresh(ticker <-chan time.Time) {
 		select {
 		case thisTime := <-ticker:
 			thisTimeInUTC := thisTime.UTC()
-			thisEndpoint := e.Endpoint + "?last-modified=" + e.lastUpdate.Format(time.RFC3339)
+
+			// Parse the endpoint url defined
+			url, urlErr := url2.Parse(e.Endpoint)
```

Comment:
Instead of aliasing the package, could you please use a different name for the variable?

Reply:
IDE imported it with an alias. Removed.